app/AndroidManifest.xml

```diff
@@ -87,6 +92,42 @@
         android:protectionLevel="dangerous">
     </permission>
 
+    <permission
+        android:name="${applicationId}.permission.READ_NFC_DATA"
```

Comment:
@avazirna Here the permission `READ_NFC_DATA` doesn't make sense since the purpose of this permission is not to give an external app permission to read nfc data (we don't have that functionality for external apps) but merely to protect the app's exposed components that should not ideally be exposed in the first place. As such we should have one single permission like `PROTECT_FALSELY_EXPOSED_COMPONENTS` and apply it to all such components.

Reply:
Makes sense having an umbrella permission for these, so this will be applied to `READ_NFC_DATA`, `WRITE_NFC_DATA`, `DRAW_BOUNDARY_AREA` and `PRINT`